tensorflow_serving/batching/batching_session.h

```diff
@@ -99,6 +99,33 @@ struct BatchingSessionOptions {
   //
   // If left empty, no rounding/padding is performed.
   std::vector<int> allowed_batch_sizes;
+
+  // If set to true, padding is performed for tensors of the same name
+  // but with unequal dimensions, so that all tensors of the same name have
+  // equal dim sizes.
+  // For each tensor its first element is used as padding value.
+  //
+  // For example:
+  // given input tensors of shapes [1, 500, 101], [2, 300, 101], [1, 400, 101]
+  // they will be padded to shapes [1, 500, 101], [2, 500, 101], [1, 500, 101].
+  //
+  // Supported tensor datatypes:
```

Comment:
Well, there is DT_BFLOAT, which is as I understand (I might be wrong) is only used internally and DT_VARIANT which was added to tensorflow about a month ago. Tfserving uses earlier version of tensorflow, so I can't add support for DT_VARIANT.